### DISCUSSION TITLE
File not found error

### DISCUSSION THREAD

**Discussion by emapiri** [simnibs/simnibs#290] [7.Creating head models]
I am very new to this and I'm trying to make a head model of a T1 weighted image but this is what I keep getting:
MacBook-Air-126:~ /Volumes/tDCS/MRI 
-bash: /Volumes/tDCS/MRI: is a directory
MacBook-Air-126:~ charm 504 MRI/504_T1.nii
[ simnibs ]INFO: simnibs version 4.0.1
[ simnibs ]INFO: charm run started: Fri Feb 23 20:09:08 2024
[ simnibs ]INFO: Reading settings from /Users/Applications/SimNIBS-4.0/simnibs_env/lib/python3.9/site-packages/simnibs/charm.ini
[ simnibs ]CRITICAL: Uncaught exception
FileNotFoundError: Could not find input T1 file: MRI/504_T1.nii

**Comment by oulap** [comment]
It seems that charm cannot find the MRI scan you passed to it.

Can you try changing the folder to the path where the MRI scan is, e.g.,

cd <path_to_mri_folder>

where <path_to_mri_folder> is the folder path in which the MRI scan is stored. Looking at the output this should most likely be:

 /Volumes/tDCS/MRI

Once you're in that folder, you can call:

charm 504 504_T1.nii
